samples/drivers/video/capture/sample.yaml

```
@@ -11,6 +11,7 @@ tests:
       - platform:mimxrt1170_evk/mimxrt1176/cm7:SHIELD="nxp_btb44_ov5640;rk055hdmipi4ma0"
       - platform:mimxrt1170_evk@B/mimxrt1176/cm7:SHIELD="nxp_btb44_ov5640;rk055hdmipi4ma0"
       - platform:frdm_mcxn947/mcxn947/cpu0:SHIELD="dvp_20pin_ov7670;lcd_par_s035_8080"
+      - platform:ek_ra8d1:SHIELD="dvp_20pin_ov7670;rtkmipilcdb00000be"
```

[blocking :x:]
It is possible that the current CI failure happens as the `sample.yaml` platform definition need the board listed with all the qualifiers. Maybe something like this but am not familiar enough with this chip to be entirely sure:

```suggestion
      - platform:ek_ra8d1/r7fa8d1bhecbd:SHIELD="dvp_20pin_ov7670;rtkmipilcdb00000be"
```